### PR TITLE
Connected back and front end of add item page

### PR DESCRIPTION
**Pull request by mols2ma**

### CODE REVIEW

File: front-end/src/components/AddItem.vue [hunk 1]
```diff
@@ -53,7 +56,7 @@
 
 <script>
 import http from "../http-common";
-import { router } from '../router';
+import ItemService from '../service/ItemService';
```
Comment (elishaaquino) [inline front-end/src/components/AddItem.vue:59]: where's the ItemService file?

File: front-end/src/components/UserAccount.vue [hunk 2]
```diff
@@ -68,11 +69,27 @@ export default {
...
+                    response => {
+                        console.log(response.data);
+                    }
+                );
             }
         },
         created() {
-            this.userUsername = this.$route.params.userUsername;
+
+            var userId = JSON.parse(localStorage.user)["id"];
```
Comment (elishaaquino) [inline front-end/src/components/UserAccount.vue:90]: is there an id to access in user in localstorage rn?

File: back-end/src/main/java/com/nearbites/controller/ItemController.java [hunk 3]
```diff
@@ -12,26 +12,28 @@
...
+    @GetMapping("/allItemsUserAccount")
     public ResponseEntity<List<Item>> getAllItems() {
         List<Item> items = new ArrayList<>();
 
         itemRepository.findAll().forEach(items::add);
         return new ResponseEntity<>(items, HttpStatus.OK);
     }
 
-    @PostMapping("/items")
+    @PostMapping("/additem")
```
Comment (maggiechang) [inline back-end/src/main/java/com/nearbites/controller/ItemController.java:29]: is the endpoint change reflected in the front end too?

File: front-end/src/components/UserAccount.vue [hunk 4]
```diff
@@ -68,11 +69,27 @@ export default {
...
         methods: {
             set_SelectedButton(value){
+
                 this.btn_id=value;
+
+            },
+
+            getItems() {
+
+                // var userId = JSON.parse(localStorage.user)["id"];
```
Comment (maggiechang) [inline front-end/src/components/UserAccount.vue:79]: delete this comment if not needed

File: front-end/src/components/AddItem.vue (hunk 1 above)
Comment (mols2ma) [reply front-end/src/components/AddItem.vue:59]: Oh man I guess I didn’t push it.  I’ll push it

File: front-end/src/components/UserAccount.vue (hunk 2 above)
Comment (mols2ma) [reply front-end/src/components/UserAccount.vue:90]: Yeah when u put the user in local storage, it has the id in it

File: back-end/src/main/java/com/nearbites/controller/ItemController.java (hunk 3 above)
Comment (mols2ma) [reply back-end/src/main/java/com/nearbites/controller/ItemController.java:29]: What do you mean?

File: front-end/src/components/UserAccount.vue (hunk 4 above)
Comment (mols2ma) [reply front-end/src/components/UserAccount.vue:79]: Okie I will make the change

**Pull request closed without merging**